code/modules/materials/Mat_Materials.dm

```diff
@@ -257,13 +446,104 @@ ABSTRACT_TYPE(/datum/material)
 			X.execute(owner, attackobj, attacker, meleeorthrow)
 		return
 
+//Material definitions
 
+ABSTRACT_TYPE(/datum/material/interpolated)
 /datum/material/interpolated
 	mat_id = "imcoderium"
 	name = "imcoderium"
 	desc = "You should not be seeing this"
 	color = "#6f00ff"
 
+	New(var/datum/material/mat1,var/datum/material/mat2,var/t)
+		..()
+		var/ot = 1 - t
```

Comment:
t? ot?

Reply:
no idea, copy pasted old code and just moved it around